pkg/policy/cell/cell.go

```diff
@@ -31,21 +31,27 @@ var Cell = cell.Module(
 )
 
 type Config struct {
-	PolicyQueueSize uint `mapstructure:"policy-queue-size"`
+	EnableWellKnownIdentities bool `mapstructure:"enable-well-known-identities"`
+	PolicyQueueSize           uint `mapstructure:"policy-queue-size"`
 }
 
 var defaultConfig = Config{
-	PolicyQueueSize: 100,
+	// EnableWellKnownIdentities is enabled by default as this is the
+	// original behavior. New default Helm templates will disable this.
+	EnableWellKnownIdentities: true,
```

Comment:
Should we add the comment as well from the `pkg/option/config.go` ? (Good to have some context maybe)
```
// EnableWellKnownIdentities enables the use of well-known identities.
// This is requires if identiy resolution is required to bring up the
// control plane, e.g. when using the managed etcd feature
```

Reply:
This should probably rather be part of a user-facing documentation, if necessary. I don't think it's very useful as a code comment as is. The usage flag already covers part of it and when installing managed etcd the respective Helm flag should already be set accordingly AFAIK. I think I'll leave this as is for now and if needed follow-up in a separate PR.